.travis.yml

```diff
@@ -5,19 +5,12 @@ os:
   - linux
 before_install:
   - chmod +x devtools/travis-ci/chimera.sh
-  - chmod +x devtools/travis-ci/miniconda.sh
   # Chimera:
   - devtools/travis-ci/chimera.sh
-  # Miniconda:
-  - devtools/travis-ci/miniconda.sh
 install:
   # Install Pychimera
-  - source $HOME/miniconda/etc/profile.d/conda.sh
-  - conda activate
-  - $HOME/miniconda/bin/conda config --set always_yes yes
-  - $HOME/miniconda/bin/conda install -c insilichem pychimera
+  - pip install pychimera
```

Comment:
This might not work if the system libraries do not match the ones expected by Chimera. It's advisable to install with conda for that very reason. I am curious why you changed that. Any problem you'd like to report?

Reply:
I think it was because it worked with pip and I saw it more simple (with this way I don't need to install miniconda in travis) and I though that maybe it was not necessary to change it to install it with conda. But I will change it if it's advisable to install it with conda.

Reply:
I have been trying for a while install pychimera with conda, but when pytest execute the test and try to import pychimera it raises an error. It seems that the import is looking for the package in the `site-packages/` directory outside the miniconda directory. How can I solve this problem?

Reply:
I have done successfully the implementation of the installation of pychimera with conda in the .travis.yml file in the last commit.